src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,55 @@
+@import './variables';
+
+@mixin product-title-font {
+  font-family: $card-ff;
+  color: $text-color;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+@mixin product-code-font {
+  font-family: $card-ff;
+  color: $text-color;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+@mixin product-code {
+  font-family: $card-ff;
+  color: $product-code-color;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+@mixin price-font {
+  font-family: $card-ff;
+  color: $price-color;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+@mixin buy-button-font {
+  font-family: $card-ff;
+  color: $buy-botton-color;
```

Comment:
There is a typo in the variable name `$buy-botton-color`, it should be `$buy-button-color` to maintain consistency and avoid potential errors when using the variable.